src/fallback.ts

```diff
@@ -333,8 +333,8 @@ export class GrpcClient {
           })
           .then(([ok, buffer]: [boolean, Buffer | ArrayBuffer]) => {
             if (!ok) {
-              const status = statusDecoder.decodeRpcStatus(buffer);
-              throw new Error(JSON.stringify(status));
+              const error = statusDecoder.decodeErrorFromBuffer(buffer);
+              throw error;
```

Comment:
What is the type of `error` in this case?  Is it `Error`, or `string`?  Kinda curious. Looking at the previous code, I wish it had done a `JSON.stringify(status, null, 2)` to make sure the result is more readable.

Reply:
it's `Error` extended with extra gRPC related fields, similar to `ServiceError` from gRPC. There is no `JSON.stringify` anymore since folks asked us to make it a proper `Error`, and this is the change this PR makes :)